.github/workflows/dormant-issues-prs.yml

```
@@ -18,10 +18,11 @@ jobs:
           days-before-stale: 180
           days-before-close: -1  # never close
           stale-issue-label: ":sleeping: Dormant"
+          stale-pr-label: ":sleeping: Dormant"
           remove-stale-when-updated: true
           stale-issue-message: >
-            Hey, there hasn't been any activity on this for more than 180
-            days, so we have marked it as "dormant" for now until there is some
+            Hey, there hasn't been any activity on this issue for more than 180
+            days. So for now, we have marked it as "dormant" until there is some
```

```suggestion
            days. For now, we have marked it as "dormant" until there is some
```
Even shorter? And I don't think we're losing any meaning... Very nice anyway!

---

Thanks. I applied it.